src/pages/repository.html

```diff
@@ -1,297 +1,272 @@
-
-						<div id="additionalParams" ng-if="repositoryInfo.type != 'master' && repositoryInfo.type != 'ontop'">
-							<div class="form-group row">
-								<label class="col-xs-3 col-sm-3 col-md-3 col-form-label">{{repositoryInfo.params.storageFolder.label}}</label>
-								<div class="col-xs-4 col-sm-4 col-md-4">
-									<input class="form-control" id="storageFolder" type="text" ng-model="repositoryInfo.params.storageFolder.value">
-								</div>
-							</div>
-							<div class="form-group row">
-								<label for="ruleset" class="col-xs-2 col-sm-2 col-md-2 col-form-label">{{repositoryInfo.params.ruleset.label}}</label>
-								<div class="col-xs-1 col-md-1">
-									<em class="icon-warning icon-2x"
-										ng-show="rulesetWarning()"
-										tooltip="{{rulesetWarning()}}"
-										tooltip-placement="top"></em>
-								</div>
-								<div class="col-xs-4 col-sm-4 col-md-4">
-									<select ng-disabled="editRepoPage" class="form-control" id="ruleset" ng-model="repositoryInfo.params.ruleset.value"
-										ng-change="repositoryInfo.params.disableSameAs.value = repositoryInfo.params.ruleset.value.indexOf('owl') === 0 ? 'false' : 'true'"
-										ng-options="r.id as r.name for r in rulesets">
-									</select>
-								</div>
-								<div class="form-group col-xs-5" ng-if="!editRepoPage">
-									<div class="btn btn-outline-primary btn-sm">
-										<div ngf-select ngf-change="uploadRuleset($files)" ngf-multiple="false"
-											 class="selectFiles pointer clearfix"
-											 accept=".pie"
-											 tooltip="{{activeLocation().local ? 'Select file' : 'Not supported for remote locations'}}"
-											 tooltip-trigger="mouseenter"
-											 ng-disabled="!activeLocation().local">
-											<em ng-hide="uploadFileLoader" class="fa fa-upload fa-2x"></em>&nbsp;
-											<div ng-show="uploadFileLoader" onto-loader size="24" class="spinner-container ot-loader"></div>
-											<span class="pageInfo" style="vertical-align: super; color: #8a8a8a;">Upload custom ruleset</span>
-										</div>
-									</div>
-									<input class="form-control" id="customRuleSet" type="file" name="ruleset" style="display: none;"/>
-								</div>
-							</div>
-							<div class="form-group row">
-								<div class="checkbox offset-xs-3 offset-md-3">
-									<label class="col-xs-6 col-md-6"
-										   tooltip="GraphDB uses a non-rule implementation of owl:sameAs, which can be enabled or disabled independently of the ruleset."
-											tooltip-placement="bottom">
-										<input id="disableSameAs" ng-disabled="editRepoPage || repositoryInfo.params.ruleset.value === 'empty'" type="checkbox" ng-true-value="'true'" ng-false-value="'false'" ng-model="repositoryInfo.params.disableSameAs.value"/>
-										{{repositoryInfo.params.disableSameAs.label}}
-									</label>
-								</div>
-							</div>
+                            </div>
```

Comment:
this template is just horrible :(

Reply:
Sorry about the look, but had to fix styling.